tests/integration/targets/mso_schema_site_anp_epg_useg_attribute/tasks/main.yml

```diff
@@ -0,0 +1,761 @@
+# Test code for the MSO modules
```

Comment:
only 4.1 seems to pass the tests, could you recheck other instances. 3.7 should at least also pass in my opinion

Reply:
I need to discuss a few items with Lionel regarding the different versions check.